### PR TITLE
PLT-890 Alter debian package naming convention

### PR DESCRIPTION
**Pull request by elopatin-plos**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.plos</groupId>
   <artifactId>ned-cas</artifactId>
-  <version>1.3.0-SNAPSHOT</version>
+  <version>1.3.0</version>
```
Comment (mbowen) [inline pom.xml]: I think this should remain -SNAPSHOT. the master branch will have fixed versions which will merge into when we do releases.

File: src/deb/control/control [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Package: [[artifactId]]
 Version: [[version]]
-Description: [[name]]
+Description: [[name]]+[[build.timestamp]]-plos[[build.counter]]
 Architecture: all
```
Comment (jonocodes) [inline src/deb/control/control]: Should this be in the version field or the description field?

Comment (elopatin-plos) [inline src/deb/control/control]: Good question. Dipro is using the Description field. However based on this reference, Version seems more applicable (as it does in general):
https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Version

Comment (elopatin-plos) [inline src/deb/control/control]: done, moved to version field

**Pull request merged**